dayrepo/snippets/templates/car_post.html

```diff
@@ -0,0 +1,26 @@
+<head>
+    {% load static %}
+    <link rel="stylesheet" href='{% static "css/cars.css" %}'>
+</head>
+<h2>車両情報設定</h2>
+<form method="post">
+    <p>
+    <div id=vehicle_number>
+        <label>車両番号:</label>
+        <input type="text"  maxlength="10" id="place_name" name="place_name" placeholder="札幌" required>
```

Comment:
特に理由がなければ、nameは基本的に不要です
```suggestion
        <input type="text"  maxlength="10" id="place_name" placeholder="札幌" required>
```